LazyJsonUndoRedo.js

```diff
@@ -426,10 +460,178 @@
 
 
 
+//--------------------------------------------------------------------------------------------------
+//  Merge
+//--------------------------------------------------------------------------------------------------
+    /**
+     * Set attributes whose change list will be merged
+     * @param obj - attributes in this object
+     * @param list - attribute list
+     */
+    p.setMergeWhitelist = function (obj, list) {
+        this.removeMergeWhitelist(obj);
+        this._mergeWhitelists.push({obj: obj, list: list});
+    };
+
+    p.getMergeWhitelist = function (obj) {
+        for (var i = 0, l = this._mergeWhitelists.length; i < l; ++i) {
+            if (this._mergeWhitelists[i].obj === obj) {
+                return this._mergeWhitelists[i];
+            }
+        }
+    };
+
+    p.removeMergeWhitelist = function (obj) {
+        for (var i = 0; i < this._mergeWhitelists.length; ++i) {
+            if (this._mergeWhitelists[i].obj === obj) {
+                this._mergeWhitelists.splice(i--, 1);
+            }
+        }
+    };
+
+    /**
+     * Set attributes whose change list will be merged
+     * @param obj - attributes in this object
+     * @param list - attribute list
+     */
```

Comment:
Sorry for the wrong comment.

It should be `Set attributes whose change list will *not* be merged`